packages/rocketchat-i18n/i18n/pt-BR.i18n.json

```diff
@@ -2720,7 +3343,7 @@
   "Random": "Aleatória",
   "RD Station": "RD Station",
   "RDStation_Token": "RD Station Token",
-  "React_when_read_only": "Permitir Reagir",
+  "React_when_read_only": "Permitir Reagir ",
```

Comment:
```suggestion
  "React_when_read_only": "Permitir Reagir",
```

Reply:
Sure! It would be a pleasure!

There are some spaces, besides those you've mentioned. Can I remove them too?

Reply:
yup, feel free to remove them too. thx